system-tests/project-fixtures/vue2-cli/src/components/HelloWorld.cy.js

```diff
@@ -9,6 +9,15 @@ describe('<Logo />', () => {
       propsData: {
         msg: slotContent,
       },
+      data () {
```

Comment:
Can we get a test for this with `vue3`

Reply:
We already have one here https://github.com/cypress-io/cypress/pull/24479/files#diff-f9a3bc77ab97b5f10f220de6ab30201be456aad25cb85e8197c52b2b90959684R24-R30

Reply:
Sorry for missing that!!